system_report.sh

```diff
@@ -49,3 +49,5 @@ echo
 echo "$ tree -L 2 /opt/android-ndk"
 tree -L 2 /opt/android-ndk
 echo
+
+ver_line="$(jq --version)" ;                      echo "* jq: $ver_line"
```

Comment:
Why there are so many whitespaces here?

Reply:
aye, you can get rid of the whitespace here :)